var/spack/repos/builtin/packages/py-click/package.py

```diff
@@ -2,16 +2,24 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.build_systems.python import PythonPackage
+from spack.directives import depends_on, version
```

Comment:
Is this from upstream? For such a common package, I prefer verbatim copies from upstream that keep the differences at a minimum. After all, I will have to resolve all merge conflicts when we redeploy.

Reply:
Thanks!